gymnasium/envs/mujoco/mujoco_env.py

```diff
@@ -288,8 +284,17 @@ def _get_viewer(
 
         return self.viewer
 
-    def get_body_com(self, body_name):
-        return self.data.get_body_xpos(body_name)
+    def close(self):
+        if self.viewer is not None:
+            self.viewer = None
+            self._viewers = {}
+
+    def viewer_setup(self):
```

Comment:
Why does this function still exist if we are using the `default_camera_config` parameter

Reply:
This function only concerns the mujoco environments that depend on `mujoco_py` which are not maintained.